projects/origin-release/ipi_deprovision/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
This file should be somewhere under `core-services`, or even better, in `ci-tools` repo.

Reply:
https://github.com/openshift/ci-tools/pull/26